src/index.html

```diff
@@ -3,13 +3,19 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"/>
+
     <title>To do List</title>
 </head>
 <body>
     <div class="list-items">
         <div class="to-do">Today's To Do 💪🏻</div>
         <div>
-            <input class="input-list" type="text" placeholder="Add to your list...">
+            <input class="input-list" type="text" maxlength="50" placeholder="Add to your list...">
         </div>
         <div class="list-container" >
             

```

Comment:
- [ ] Checked items are not removed from the list/ local storage when the _Clear all completed_ button is clicked. Kindly ensure to improve upon this aspect